### PR TITLE
FAT-2628/C347866

### PR DESCRIPTION
**Pull request by re-jas**
- [C347866: Page request + Cancel request triggers](https://issues.folio.org/browse/FAT-2628)
- [Allure](https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/2228/allure/)

### CODE REVIEW

File: cypress/e2e/recieve-notice/page-request-cancel-request-triggers.cy.js [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+        .then(() => {
+          UserEdit.addServicePointViaApi(
+            testData.userServicePoint.id,
+            userData.userId,
+            testData.userServicePoint.id
+          );
+
+          cy.getCirculationRules().then((response) => {
+            testData.baseRules = response.rulesAsText;
+            testData.ruleProps = CirculationRules.getRuleProps(response.rulesAsText);
```
Comment (penfriend) [inline cypress/e2e/recieve-notice/page-request-cancel-request-triggers.cy.js:152]: let's save baseRules right before adding the new rule

File: cypress/e2e/recieve-notice/page-request-cancel-request-triggers.cy.js [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+    barcode: generateItemBarcode(),
+    title: `Instance ${getRandomPostfix()}`,
+  };
+  const testData = {
+    userServicePoint: ServicePoints.getDefaultServicePointWithPickUpLocation('autotestReceiveNotice', uuid()),
+    ruleProps: {},
+  };
+  const createNoticeTemplate = (noticeName) => {
+    return {
+      name: `Autotest_${getRandomPostfix()}_${noticeName}`,
```
Comment (penfriend) [inline cypress/e2e/recieve-notice/page-request-cancel-request-triggers.cy.js:41]: let's use the method getTestEntityValue(Value) from utils/stringTools.js

File: cypress/e2e/recieve-notice/page-request-cancel-request-triggers.cy.js [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+      description: 'Created by autotest team',
+      category: 'Request',
+      subject: `Autotest_${getRandomPostfix()}_${noticeName}`,
```
Comment (penfriend) [inline cypress/e2e/recieve-notice/page-request-cancel-request-triggers.cy.js:44]: let's use the method getTestEntityValue(Value) from utils/stringTools.js

File: cypress/e2e/recieve-notice/page-request-cancel-request-triggers.cy.js [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+    };
+  };
+  const checkNoticeIsSent = (checkParams) => {
+    SearchPane.searchByUserBarcode(userData.barcode);
+    SearchPane.findResultRowIndexByContent(checkParams.desc).then((rowIndex) => {
+      SearchPane.checkResultSearch(checkParams, rowIndex);
+    });
+  };
+  const noticePolicy = {
+    name: `Autotest ${getRandomPostfix()} Overdue fine, returned`,
```
Comment (penfriend) [inline cypress/e2e/recieve-notice/page-request-cancel-request-triggers.cy.js:71]: let's use the method getTestEntityValue(Value) from utils/stringTools.js

File: cypress/e2e/recieve-notice/page-request-cancel-request-triggers.cy.js (hunk 1 above)
Comment (re-jas) [reply cypress/e2e/recieve-notice/page-request-cancel-request-triggers.cy.js:152]: done

File: cypress/e2e/recieve-notice/page-request-cancel-request-triggers.cy.js (hunk 2 above)
Comment (re-jas) [reply cypress/e2e/recieve-notice/page-request-cancel-request-triggers.cy.js:41]: done

File: cypress/e2e/recieve-notice/page-request-cancel-request-triggers.cy.js (hunk 3 above)
Comment (re-jas) [reply cypress/e2e/recieve-notice/page-request-cancel-request-triggers.cy.js:44]: done

File: cypress/e2e/recieve-notice/page-request-cancel-request-triggers.cy.js (hunk 4 above)
Comment (re-jas) [reply cypress/e2e/recieve-notice/page-request-cancel-request-triggers.cy.js:71]: done

**Review by penfriend — APPROVED** [review]

**Pull request merged**